### DISCUSSION TITLE
Forward messages to specific mail server and port

### DISCUSSION THREAD

**Discussion by EthraZa** [simple-login/app#677] [Q&A] [answered]
Their is a way or is it planned to be implemented at some point, the possibility of forwarding emails to a specific MX server at a specific port?
For instance, I could have a domain or alias that I would like to delivery its emails to [mail.domain.com:2525](url).

**Comment by nguyenkims** [comment, marked as answer]
We don't have any plan to add this feature yet.

**Reply by EthraZa** [reply]
I see. I asked because when I was looking into migrate my infra to DigitalOcean, they have their port 25 closed and any email traffic must go through port 2525. So if I decide to go the DigitalOcean path, I would need to find another service to route my emails since I have this need.

Do you guys have some sort of bounty project for feature development so we can choose to pay for specific features and help to envolve the project?

**Reply by nguyenkims** [reply]
Oh I see, you are referring to the self hosted SimpleLogin then. Cloud services like DigitalOcean aren't good for hosting emails as they usually block port 25 and their IP addresses have a low reputation (probably because of their popularity). I'd suggest using  smaller cloud services or have a high entry points that don't have these issues. We are using UpCloud and are quite happy with their services.

> Do you guys have some sort of bounty project for feature development so we can choose to pay for specific features and help to envolve the project?

We don't have a specific bounty program but you can vote on the features you want to arrive and we prioritise the features based on the number of votes (among other criteria).

**Reply by EthraZa** [reply]
I was thinking about route mail from SL.io to mx.zoho.com:25 OR mail.mypostfix:2525.
But I got it. Thanks
